project/Dependencies.scala

```diff
@@ -124,7 +127,8 @@ object Dependencies {
 
   lazy val httpTestkit = l ++= Seq(
     Test.junit, Test.junitIntf, Compile.junit % "provided",
-    Test.scalatest.value.copy(configurations = Some("provided; test"))
+    Test.scalatest.value.copy(configurations = Some("provided; test")),
+    Test.specs2.value
```

Comment:
This will work as long as we don't want to use specs2 inside the `docs` project since it uses this dependency mapping `httpTestkit % "compile;test->test"`.